index.tt

```diff
@@ -6,152 +6,251 @@
+    <p class="text-center lead">
+      The Nix Packages collection (<a href="https://github/NixOS/nixpkgs">nixpkgs</a>) is a set of
+      <strong>over 60 000 packages</strong> for the Nix package manager,
+      released under a permissive <a href="https://spdx.org/licenses/MIT.html">MIT/X11 license</a>.
+    </p>
+    <form>
+      <div class="input-append">
+        <input class="input-xxlarge" placeholder="Search for packages" type="text" />
+        <button class="btn">Search</button>
+      </div>
+      <p class="text-center">or <a href="[% root %]nixos/options.html">search among many NixOS options.</a></p>
+    </form>
   </div>
+</div>
+
+<hr />
+
+<h1 class="text-center">Examples...</h1>
+
+<div id="features">
+
+  <section>
+    <div class="row-fluid">
+      <div class="span6">
+
+        <h2>Try new tools without fear</h2>
+        <p>Don't clutter your system with tools that you use only now and then.</p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command">python --version</span>
+python: command not found
+<span class="nix-command"><strong>nix-shell -p python3</strong></span>
+(nix-shell)<span class="nix-command">python --version</span>
+Python 3.7.7</pre>
+
+        <h2>One tool, multiple languages</h2>
+        <p></p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command"><strong>nix-shell -p python3 nodejs go rustc</strong></span>
+(nix-shell)<span class="nix-command">node --version</span>
+v10.20.1
+(nix-shell)<span class="nix-command">go version</span>
+go version go1.14.1 linux/amd64
+(nix-shell)<span class="nix-command">rustc --version</span>
+rustc 1.42.0</pre>
+
+      </div>
+      <div class="span6">
 
+        <h2>Isolated development environments</h2>
+        <p>
+          After you get familiar with <code>nix-shell -p</code> you can take
+          the next step further and <a href="[%root%]learn.html">learn some
+            Nix</a>. To setup a more persistent environment you can also write a
+          simple <code>shell.nix</code> file:
+        </p>
+        <pre class="well" style="margin-top:1em">
+{ pkgs ? import &lt;nixpkgs&gt; {}
+}:
+pkgs.mkShell {
+  name = "dev-shell";
+  buildInputs = [
+    pkgs.python3
+    pkgs.python3Packages.virtualenv
+    pkgs.nodejs
+    pkgs.yarn
+  ];
+}</pre>
+        <p>Then enter development environment with:</p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command"><strong>nix-shell</strong></span>
+(nix-shell)<span class="nix-command">virtualenv --version</span>
+16.7.9
+(nix-shell)<span class="nix-command">yarn --version</span>
+1.22.4</pre>
+        <p>
+          Commit the above <code>shell.nix</code> file and let you coworkers have
+          easier time setting their development environment.
+        </p>
+
+      </div>
+    </div>
+  </section>
+  <section class="tab-pane" id="build">
+    <div class="row-fluid">
+      <div class="span6">
+        <h2>Minimal docker image</h2>
```

Comment:
What I forgot to add to this is example is explaining in short why Nix is much better at composing docker images.
eg.
1) no build tools present in you final docker image.
2) no need to remember to delete cache of tools used to produce image (`apt-get clean`)
3) smart automatic layers

Of-course we should also end up pointing to Nix+Docker guide once it exists.

Reply:
yes, important point. "**WHY** would i care about this?"